nixtla/nixtla_client.py

```diff
@@ -658,6 +667,20 @@ def _maybe_assign_weights(
         else:
             self.weights_x = type(df)({"features": x_cols, "weights": weights})
 
+    def _maybe_assign_feature_contributions(
+        self,
+        feature_contributions: Optional[list[list[float]]],
```

Comment:
This syntax was introduced in python 3.9, while we still support 3.8 we have to use `typing.List`. Please also change the weights method signature to use that.
```suggestion
        feature_contributions: Optional[List[List[float]]],
```